cmd/lncli/cmd_open_channel.go

```diff
@@ -635,6 +646,60 @@ func printChanPending(update *lnrpc.OpenStatusUpdate_ChanPending) error {
 	return nil
 }
 
+// readTerminalOrFile reads a single line from the terminal. If the line read is
+// short enough to be a file and a file with that exact name exists, the content
+// of that file is read and returned as a string. If the content is longer or no
+// file exists, the string read from the terminal is returned directly. This
+// function can be used to circumvent the N_TTY_BUF_SIZE kernel parameter that
+// prevents pasting more than 4096 characters (on most systems) into a terminal.
+func readTerminalOrFile(quit chan struct{}) (string, error) {
+	maybeFile, err := readLine(quit)
+	if err != nil {
+		return "", err
+	}
+
+	// Absolute file paths normally can't be longer than 255 characters so
+	// we don't even check if it's a file in that case.
+	if len(maybeFile) > 255 {
```

Comment:
it's possible for a psbt to be <= 255 right?

Reply:
Yes. That's why we return the string directly if a file with that name doesn't exist.